allennlp/models/coreference_resolution/coref.py

```diff
@@ -53,6 +53,10 @@ class CoreferenceResolver(Model):
         spans we retain with respect to the number of words in the document.
     max_antecedents: int, required.
         For each mention which survives the pruning stage, we consider this many antecedents.
+    coarse_to_fine: bool, optional (default = False)
+        Whether or not to apply the coarse-to-fine filtering.
+    num_coref_layers: int, optional (default = 1)
```

Comment:
`num_span_refinement_layers` might be a better name, given your description 😄

Reply:
Yeah my description wasn't 100% accurate either. Updated both.